asr_ai3/addons/skills/fnc_setUnitCamo.sqf

```diff
@@ -13,7 +13,7 @@ private _gearItems = [_uniform, _vest, _ruck]; //uni must be checked first for g
 
 //Don't do anything if the relevant items didn't change
 if (_unit getVariable ["asr_ai_skills_camoCache", []] isEqualTo [_gearItems, _tube]) exitWith {};
-unit setVariable ["asr_ai_skills_camoCache", [_gearItems, _tube]];
+_unit setVariable ["asr_ai_skills_camoCache", [_gearItems, _tube], false];

```

Comment:
The `,false` at the end is not required. And will make the script slower. Considering that this PR's whole aim is to make that script faster... That's not good.

Stupid oversight on `unit` by me...